content/about/releases.fr.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Releases
```

Comment:
Just a quick typo: title should be `releases` and not `Releases`, I know it's minimal, but it's just to guarantee, your file metadata matches the english one!